### PR TITLE
Save local backup when using AccountUpdateTool

### PR DESCRIPTION
**Pull request by cgtz**
Allow the user to set up a backup directory path so that the state
before performing an account metadata update is saved.

### CODE REVIEW

File: ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java [hunk 1]
```diff
@@ -159,6 +159,13 @@ public static void main(String args[]) throws Exception {
         .ofType(Short.class)
         .defaultsTo(Container.getCurrentJsonVersion());
 
+    ArgumentAcceptingOptionSpec<String> backupDirOpt = parser.accepts("containerSchemaVersion",
+        "Optional local backup directory path. Defaults to /tmp/account-update-tool-backups.")
+        .withRequiredArg()
+        .describedAs("backup_dir")
+        .ofType(String.class)
+        .defaultsTo("/tmp/account-update-tool-backups");
```
Comment (jsjtzyy) [inline ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java]: please use constants for this default directory path.

File: ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java [hunk 2]
```diff
@@ -235,6 +245,7 @@ private static AccountService getHelixAccountService(String zkServer, String sto
         String.valueOf(zkSessionTimeoutMs));
     helixConfigProps.setProperty(HelixAccountServiceConfig.ZK_CLIENT_CONNECT_STRING_KEY, zkServer);
     helixConfigProps.setProperty(HelixPropertyStoreConfig.HELIX_PROPERTY_STORE_PREFIX + "root.path", storePath);
+    helixConfigProps.setProperty(HelixAccountServiceConfig.BACKUP_DIRECTORY_KEY, backupDir);
```
Comment (jsjtzyy) [inline ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java]: Do we need to specify default `backDir` in `HelixAccountServiceConfig`? Currently, the default value is empty string.

Comment (cgtz) [reply ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java]: This feature is disabled by default. That is why the default is empty string

**Comment by jsjtzyy** [comment]
lets merge this tool first.

**Pull request merged**